css/dashboard/helpers.less

```diff
@@ -7,11 +7,14 @@
 @student-name-width: 150px;
 @border-width: 1px;
 @cell-border: @border-width solid @main-dark;
-@row-height: 25px;
+@row-height: 40px;
 @row-padding: 8px;
 @row-outer-height: @row-height + 2 * @row-padding + 2 * @border-width;
 @expanded-student-row-height: 200px;
 @expanded-question-prompt-max-height: 100px;
+@expanded-question-width: 350px;
+@closed-question-width: 50px;
```

Comment:
These values seem to be defined both in CSS and JS now. I was trying to avoid that. 
I think that collapsed and expanded could be done by defining `flex: 1` (collapsed) and `flex: 100` (expanded). It should also help with cases when there are not many questions in a given activity. Then we wouldn't have to have these dimensions here too.
Otherwise, I've seen webpack LESS variables loader which lets you load values from this feel and have them available in JS. Although, it seems pretty exotic, so I'd rather try to avoid it.

Reply:
I was reluctant to do this, but I wasn't able to align columns correctly otherwise. I may try one last attempt with flex again or have a look at the variable loader you mention.

Reply:
I think I addressed these issues in in 62e7493